compile/compile_test.go

```diff
@@ -1988,19 +2000,20 @@ func TestCompilerWasmTargetMultipleEntrypoints(t *testing.T) {
 	files := map[string]string{
 		"test.rego": `package test
 
-		p = true`,
+		p := true`,
 		"policy.rego": `package policy
 
-		authz = true`,
+		authz := true`,
 		"mask.rego": `package system.log
 
-		mask["/input/password"]`,
+		mask contains "/input/password"`,
```

Comment:
Why not use `rego.v1` here and then the default rego version?

Reply:
I hadn't added `ast.DefaultRegoVersion` yet when I was updating this test. Have fixed it now.